spec/javascripts/govuk_publishing_components/analytics-ga4/gtm-click-tracking.spec.js

```diff
@@ -152,6 +153,7 @@ describe('Google Tag Manager click tracking', function () {
           text: 'some text'
         }
       }
+      element.setAttribute('data-module', 'gem-accordion govuk-accordion')
```

Comment:
Is it necessary to set this again? It's already done in the `beforeEach` function.

Reply:
This was to test both orders of `data-module` - it looks very similar to the beforeEach! Is this OK?